Lib/test/test_collections.py

```diff
@@ -411,6 +411,12 @@ def test_field_doc_reuse(self):
         self.assertIs(P.m.__doc__, Q.o.__doc__)
         self.assertIs(P.n.__doc__, Q.p.__doc__)
 
+    @support.cpython_only
+    def test_field_repr(self):
+        Point = namedtuple('Point', 'x y')
+        self.assertEqual(repr(Point.x), "_tuplegetter(0, 'Alias for field number 0')")
+        self.assertEqual(repr(Point.y), "_tuplegetter(1, 'Alias for field number 1')")
```

Comment:
Please add a test for a custom docstring as well:

     Point.x.__doc__ = 'abscissa'    ⟼ _tuplegetter(0, 'abscissa'')

Reply:
Thanks for catching this!